src/eeg.jl

```diff
@@ -25,23 +26,34 @@ Attributes:
 Otherwise the recipe just uses the [`topoplot`](@ref) defaults and passes through the attributes.
 """
 eeg_topoplot
-
+ function eeg_topoplot(data,labels;kwargs...) 
+    @warn "labels as positional arguments have been deprecated. Please provide them as keyword arguments"
+    eeg_topoplot(data;labels=labels,kwargs...)
+ end
+ function eeg_topoplot!(fig, data,labels;kwargs...) 
+    @warn "labels as positional arguments have been deprecated. Please provide them as keyword arguments"
+    eeg_topoplot!(fig,data;labels=labels,kwargs...)
+ end

```

Comment:
this is nice, but checkout the `@deprecate` macro for a convenient way of defining these and providing the deprecation warning 😄